src/views/Page404/mascot.svg

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Where did this file come from?
I haven't seen Gordon use a lion that looks like this before.  Did you consider using the "rampant lion"?
I'm not sure where to find it, but I did find this: 
https://i.turner.ncaa.com/sites/default/files/images/logos/schools/bgd/gordon.svg
It's white-on-white, so easier to see here: 
https://www.ncaa.com/schools/gordon
We could ask Stephen Dagley (Gordon's webmaster) for a good image.